source/helpers.coffee

```diff
@@ -6,12 +6,19 @@ defineTypeWithFields = (BaseType, namespace, className, fields) ->
   SubType = BaseType.extend(classPath)
   SubType::fields = -> _.extend BaseType::fields(), fields
   Space.resolvePath(namespace)[className] = SubType
+  definedSerializable = {}
+  definedSerializable[classPath] = SubType
+  return definedSerializable
```

Comment:
Thanks for the comments. I will improve the code if this is the best way to use messaging helper with ES6 modules:
`definedSerializable[classPath]`

![screen shot 2016-03-19 at 00 37 02](https://cloud.githubusercontent.com/assets/4280521/13894654/d58bbfea-ed6a-11e5-9feb-6f3b3b7dbbfb.png)